TicTacToe/TicTacToe.js

```diff
@@ -113,14 +103,14 @@ let game = new Game();
 let moves = document.getElementById("moves");
 let messages = document.getElementById("messages");
 let reset = document.getElementById("newGame");
-let buttons = new Array();
+let buttons = [];
 makeButtons();
-buttons.forEach(element => element.innerHTML = ".");
 
+// making the buttons and setting them to the game
 function makeButtons() {
     for (let i = 0; i < 3; i++) {
         for (let j = 0; j < 3; j++) {
-            let button = document.getElementById(i + "" + j);
+            let button = document.getElementById(`${i}${j}`);
```

Comment:
To avoid nesting too deeply, you may want to declare the callback in `button.addEventListener` as a top-level function (e.g., name it `gameStep()` or something like that). This avoids [callback hell](http://callbackhell.com/).